Source/Scene/Model.js

```diff
@@ -823,6 +835,7 @@ define([
      * @param {Matrix4} [options.modelMatrix=Matrix4.IDENTITY] The 4x4 transformation matrix that transforms the model from model to world coordinates.
      * @param {Number} [options.scale=1.0] A uniform scale applied to this model.
      * @param {Number} [options.minimumPixelSize=0.0] The approximate minimum pixel size of the model regardless of zoom.
+     * @param {Number} [options.maxiumumScale] The maximum scale for the model.
```

Comment:
Does it really make sense to specify units? It's a scale, not a measurement.
e.g. `10 meters * scale`, not `10 meters * scale meters`